private/sass/addons/_faq.scss

```diff
@@ -0,0 +1,17 @@
+//######################################################################################################################
+// #FAQ#
+
+// adds faq style for /faq/
+
+// custom title for right navigation
+.aldryn-faq-sidebar {
+    h3 {
+        @extend .text-uppercase;
```

Comment:
I created a separate issue: https://github.com/divio/django-cms-demo/issues/30
We also have to avoid duplicated css code (as this is from newsblog